atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/keyvalue/impl/DynamicColumnTable.java

```diff
@@ -30,7 +30,7 @@
     }};
 
     public static TableReference reference() {
-        return TableReference.create(Namespace.DEFAULT_NAMESPACE, "dynamicColumnsTable");
+        return TableReference.create(Namespace.DEFAULT_NAMESPACE, "dynamicColsTable");
     }
```

Comment:
I thought this was within those limits, but I see that maybe will all the prefixing and namespacing this may hit the 30-some character limit.  I assume that we are trying to get this PR in first and then add the hashing later to support all table names.